Gui/opensim/CMakeLists.txt

```diff
@@ -143,3 +143,24 @@ if (APPLE)
                   COMMENT "Build OpenSim.app.")
     add_dependencies(PrepareInstaller opensim-models)
 endif()
+
+if (NOT APPLE AND NOT WIN32)
+  if (OpenSim_FOUND)
+        set(PLATFORM_PRIVATE_PROPERTIES_FILE
+            "${CMAKE_CURRENT_SOURCE_DIR}/nbproject/private/platform-private.properties")
+        if (NOT EXISTS ${PLATFORM_PRIVATE_PROPERTIES_FILE})
```

Comment:
Typically "private" files are machine dependent and not under source control so we try to avoid them, but maybe ok in this case. I think we pass command line options for windows and mac but I'm open to change, would be good to make builds on all platforms consistent.

Reply:
This was directly copied from the Mac build rules; I don't recall off the top of my head whether it is necessary to build/run or not. I will look into it.

Reply:
Ah okay maybe this is fine then but there is no need to duplicate it if we already use a similar block for Mac.

Reply:
The block for Mac is tucked in the `IF (APPLE)` conditional, so we need to duplicate it in the Unix conditional.

Reply:
Thanks for the explanation @halleysfifthinc, I see that the two blocks for Mac and Linux are pretty similar indeed, would it be possible to combine them for easier maintenance down the line (as suggested by @chrisdembia?)

Reply:
We could use a CMake macro https://cmake.org/cmake/help/v3.0/command/macro.html